### ISSUE TITLE
Expand instructions for the Full Accessibility & 508 Office Audit

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#32363]
## Issue Description

This ticket will include identifying and documenting the steps to performing an accessibility audit. The steps included will be brief and not include a full list of responsibilities but rather a high level overview of what is expected from the accessibility reviewer when reviewing a product. The steps should be documented in Confluence under [Full Accessibility & 508 Office Audit] (https://vfs.atlassian.net/wiki/spaces/PPS/pages/1748828191/Full+Accessibility+508+Office+Audit).

---
## Tasks
- [x] Schedule time with Shira to "pick her brain" on current Full Accessibility & 508 office audit
- [x] Sean to reach out to Noah to request write-up time
- [x] Noah to write high level overview of how to perform a full accessibility & 508 office audit
- [x] Peer Review/content review performed by Jim
- [ ] Clarify comments with Noah
- [ ] Sarah and Brian review documentation
- [ ] Rewrite last couple of paragraphs

## Acceptance Criteria
- [x] We have documented the process of the Full Accessibility & 508 Office Audit in cc touchpoint in the Platform Products & Service space on Confluence.
- [x] Anyone from Platform is able to complete the Full Accessibility & 508 Office Audit

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by seanasewell** [comment]
Status update: @allison0034 and I met with @shiragoodman to discuss current full accessibility and 508 office audit process. I reached out to @noahgelman to write high level overview of process. He is expecting to have that completed by EOD Tuesday (11/23).

**Comment by noahgelman** [comment]
@seanasewell @shiragoodman 
V1 of the doc. https://vfs.atlassian.net/wiki/spaces/PPS/pages/1893662799/Full+Accessibility+508+Office+Audit+-+Moderator+Responsibilities
If people feel it's on the right track I'll go in and add some additional helpful links and screen shots

**Comment by shiragoodman** [comment]
@noahgelman thank you! I will take a look today. Is this the process you follow for a Staging Review AND a Full Accessibility & 508 Office Audit?

**Comment by seanasewell** [comment]
@shiragoodman Here is the write-up from Noah.
https://vfs.atlassian.net/wiki/spaces/PPS/pages/1893662799/Full+Accessibility+508+Office+Audit+-+Moderator+Responsibilities

**Comment by seanasewell** [comment]
@jimuhh https://vfs.atlassian.net/wiki/spaces/PPS/pages/2002059367/Steps+of+an+Accessibility+Audit

**Comment by jimuhh** [comment]
@noahgelman I will look into that page and add info from it to [my draft version](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2007433394/Steps+of+an+Accessibility+Audit-+Jim+Draft+edits)

**Comment by jimuhh** [comment]
@noahgelman -- I built out the Axe plugin instructions. Wondering if accessibility team members will have the paid versio of plugin installed or will they need to get it?

**Comment by noahgelman** [comment]
We don't use the paid version

**Comment by jimuhh** [comment]
@shiragoodman -- can we reopen this? I did some rewrites and edits before the holiday i[n this draft document ](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2007433394/Steps+of+an+Accessibility+Audit-+Jim+Draft+edits)but still have some inline comment questions I'd like to run by @noahgelman

**Comment by shiragoodman** [comment]
sure. I reopened it. I had thought the work was complete.

**Comment by jimuhh** [comment]
@Saraharaba8 and @shiragoodman -- I have finished the draft of this page. Please review it to make sure I got things right.

**Comment by shiragoodman** [comment]
@jimuhh can you link the draft page please?

**Comment by jimuhh** [comment]
Whoops. [Here you go. ](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2007433394/Steps+of+an+Accessibility+Audit-+Jim+Draft+edits)
